src/components/elements/checkbox.js

```diff
@@ -48,10 +49,10 @@ const StyledCheckbox = styled.div`
     }
```

Comment:
For Line 42-44, you can add that conditionally styling in the `Icon` component itself on line 11.

Reply:
For Line 36-38, a suggestion and a more concise way you could write that would be in:

`border: ${(props) => props.secondary && prop.checked ?  `2px solid var(--color-red)` : '2px solid var(--color-grey-5)'}`